website/content/en/docs/concepts/provisioners.md

```diff
@@ -298,7 +301,7 @@ spec:
 
 You can specify the container runtime to be either `dockerd` or `containerd`. By default, `containerd` is used.
 
-* `containerd` is the only valid container runtime when using the Bottlerocket AMI Family or when using the AL2 AMI Family and K8s version 1.24+
+* `containerd` is the only valid container runtime when using Kubernetes version 1.24+ and an AMI Family of `Bottlerocket`, `AL2`, `Windows2019`, or `Windows2022`
```

Comment:
```suggestion
* `containerd` is the only valid container runtime when using the `Bottlerocket` AMIFamily or when using Kubernetes version 1.24+ and the `AL2`, `Windows2019`, or `Windows2022` AMIFamilies.
```

The way this was updated is not fully accurate anymore. `containerd` is the only supported container runtime on `Bottlerocket` or on `AL2`, `Windows2019`, or `Windows2022` if the K8s version is past 1.24